Maestro/subscriptions.json

```diff
@@ -142,54 +90,43 @@
             "/p:NotifyGitHubUsers=dotnet/coreclr-contrib",
             "/verbosity:Normal"
           ]
-        },
-        // This handler will bring the Latest CoreFX master build into WCF master
-        {
-          "maestroAction": "wcf-general",
-          "maestroDelay": "00:10:00",
-          "ScriptFileName": "build-managed.cmd",
-          "Arguments": [
-            "--",
-            "/t:UpdateDependenciesAndSubmitPullRequest",
-            "/p:GitHubUser=dotnet-bot",
-            "/p:GitHubEmail=dotnet-bot@microsoft.com",
-            "/p:GitHubAuthToken=`$(`$Secrets['DotNetBotGitHubPassword'])",
-            "/p:ProjectRepoOwner=dotnet",
-            "/p:ProjectRepoName=wcf",
-            "/p:ProjectRepoBranch=master",
-            "/p:NotifyGitHubUsers=dotnet/wcf-contrib",
-            "/verbosity:Normal"
-          ]
         }
-      ]
+      }
     },
+    // Update dependencies in CoreCLR release/1.0.0
```

Comment:
Ah, I see. Because of how the update-dependencies functionality was implemented in the release 1.0.0 branches, each 1.0.0 subscription can only update one dependency, unlike the 1.1.0 subscriptions.  I agree that it does make sense to specify "ProjectK TFS dependencies". I've made that change.